include/triton/core/tritonserver.h

```diff
@@ -1071,6 +1071,20 @@ TRITONSERVER_DECLSPEC TRITONSERVER_Error* TRITONSERVER_InferenceRequestAddInput(
     const TRITONSERVER_DataType datatype, const int64_t* shape,
     uint64_t dim_count);
 
+/// Add a raw input to a request. The model input name, data type and shape of
+/// the input will be deduced from model configuration, this function must at
+/// most be called once on request with no other input to ensure the deduction
+/// is accurate.
+///
+/// \param inference_request The request object.
+/// \param name The name of the input. This name is only used as a reference
+/// of the raw input in other Tritonserver APIs, it doesn't assoicate with the
```

Comment:
Nit: Would be helpful for readability to break this into two sentences. "This name is used as a reference
of the raw input in other Tritonserver APIs. It is not associated with the name used in the model."